variables.tf

```diff
@@ -3,35 +3,41 @@
 #######################
 
 variable "cluster_name" {
-  type = string
+  description = "Name given to the cluster. This value will be used to generate the URLs in order to create the ingresses to access the application."
+  type        = string
 }
 
 variable "base_domain" {
-  type = string
+  description = "Base domain of the cluster. This value will be used to generate the URLs in order to create the ingresses to access the application."
+  type        = string
 }
 
 variable "argocd_namespace" {
-  type = string
+  description = "Namespace used by Argo CD where the Application and AppProject resources should be created."
+  type        = string
 }
 
 variable "cluster_issuer" {
-  type    = string
-  default = "ca-issuer"
+  description = "SSL certificate issuer to use. Usually you would configure this value as `letsencrypt-staging` or `letsencrypt-prod` on your root `*.tf` files."
+  type        = string
+  default     = "ca-issuer"
 }
 
 variable "namespace" {
-  type    = string
-  default = "thanos"
+  description = "The namespace where the application's resources will reside (it will be created in case it dows not already exist)."
+  type        = string
+  default     = "thanos"
 }
 
 variable "helm_values" {
-  description = "Helm values, passed as a list of HCL structures."
+  description = "Helm values, passed as a list of HCL structures. These values are concatenated with the default ones and then passed to the application's charts."
```

Comment:
"Helm chart value overrides. They should be passed as a list of HCL structures.` maybe ?